micro-rdk/src/lib.rs

```diff
@@ -43,36 +43,34 @@ pub mod google {
 
 /// gRPC prototypes from definitions in [api repository](https://github.com/viamrobotics/api/tree/main/proto/viam), auto-generated
 pub mod proto {
+
```

Comment:
@npmenard - I had a newish nightly rust installed, and clippy started throwing some new complaints about the generated proto code. I didn't see much utility in needing to individually suppress in each instance as we were doing before, so I just took a big hammer to clippy w.r.t. generated code. Please let me know if you dislike that choice and I can go the other way.

Reply:
That's my take, that there isn't much point to clippy for generated code, and we will just end up bouncing back and forth between needing to add and remove specific suppressions as clippy and prost each advance versions. If you still want a ticket let me know and I'll file one, but I think I'm going to merge this as is.